vendor/github.com/cloudflare/cloudflare-go/spectrum.go

```diff
@@ -20,6 +20,7 @@ type SpectrumApplication struct {
 	IPFirewall    bool                          `json:"ip_firewall,omitempty"`
 	ProxyProtocol bool                          `json:"proxy_protocol,omitempty"`
 	TLS           string                        `json:"tls,omitempty"`
+	TrafficType   string                        `json:"traffic_type,omitempty"`
```

Comment:
Please PR this in https://github.com/cloudflare/cloudflare-go/blob/master/spectrum.go
We'll vendor the change back here in separate PR.

Reply:
https://github.com/cloudflare/cloudflare-go/pull/350